app/messages.py

```diff
@@ -1,3 +1,4 @@
+from app.api.validations.user import NAME_MIN_LENGTH, NAME_MAX_LENGTH
```

Comment:
nandini45 Member Pending
@isabelcosta just a opinion if we have to shorten the code maybe instead of
writing
import NAME_MIN_LENGTH, NAME_MAX_LENGTH
we can just write
import *

Reply:
It's not that long of a line and not always short code equals good code. In this way you are only importing what you need to us @nandini45  Let me know if what you think